### PR TITLE
Add sfinv.get_page()

### PR DESCRIPTION
**Pull request by rubenwardy**

### CODE REVIEW

**Comment by rubenwardy** [comment]
This should probably return the table of the page, in fact. Or be renamed to get_page_name

**Comment by paramat** [comment]
Is this the feature @Ezhh requested recently?

**Comment by rubenwardy** [comment]
Err, no?

File: game_api.txt [hunk 1]
```diff
@@ -569,6 +569,7 @@ sfinv API by its author: https://rubenwardy.com/minetest_modding_book/en/chapter
 **Pages**
 
 * sfinv.set_page(player, pagename) - changes the page
+* sfinv.get_page(player) - get the current page. Will always return a value
```
Comment (SmallJoker) [inline game_api.txt]: All functions kinda return a value. More specific: `Will always return a (valid?) page name`

**Pull request merged**